.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
   - npm install --quiet -g gulp
+  - cd demo && npm install && cd ..
```

Comment:
Do we need to duplicate `node_modules`? I was under the impression that node module resolution works in the way that it would traverse folders up till it finds `node_modules`. I would _love_ to avoid duplication of package.json and node_modules.